package.json

```diff
@@ -32,7 +32,7 @@
     "compile": "tsc",
     "lint": "eslint",
     "typecheck": "tsc --noEmit",
-    "prepare": "npm run generate-style-spec"
+    "prepare": "npm run generate-style-spec && npm run generate-typings && npm run build"
```

Comment:
Is this needed?

Reply:
I had to add this to be able to test maplibre-gl PR. I will remove it before merging

Reply:
This is the only comment preventing it from being merged as far as I can tell, unless you would like to add more commits...?

Reply:
I reverted this change ;)